fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -129,3 +131,13 @@ def predict(self, X, *, sensitive_features, random_state=None):
         positive_probs = self._pmf_predict(
             X, sensitive_features=sensitive_features)[:, 1]
         return (positive_probs >= random_state.rand(len(positive_probs))) * 1
+
+    def _print_thresholds(self):
```

Comment:
Do we want to comment this function before we merge the change? It sticks out a bit in the code base because all the other functions in the file have a docstring.

Reply:
Did this in a follow-up commit, and made it "public" by removing the underscore since Adrin commented elsewhere that we shouldn't use private methods in the user guide.